lib/rules/no-import-of-scoped-modules-during-ssr.js

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+'use strict';
+
+const { docUrl } = require('../util/doc-url');
+
+const formFactorModule = '@salesforce/client/formFactor';
+const featureFlagModule = '@salesforce/featureFlag';
```

Comment:
The feature flag module needs a flag name at the end like this: `@salesforce/featureFlag/{flagName}`. Similar to [the `@salesforce/label/{labelReference}` scoped module](https://developer.salesforce.com/docs/platform/lwc/guide/create-labels.html)

Reply:
From the rule implementation perspective `@salesforce/featureFlag` is no different than the rest of the user-specific scoped modules

Reply:
@lpomerleau , same here. I have a conditional block where I check it an import starts with featureFlagModule. Let me know if you have any concerns regrading that.